lib/view/blocs/qr_code_bloc/qr_code_bloc.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:equatable/equatable.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:superformula_test/domain/use_cases/qr_code_get_seed_use_case.dart';
+import 'package:superformula_test/view/mappers/qr_code_view_model_mapper.dart';
+import 'package:superformula_test/view/models/qr_code_view_model.dart';
+
+part 'qr_code_state.dart';
+
+class QRCodeCubit extends Cubit<QRCodeState> {
+  QRCodeCubit(this.useCase) : super(const QRCodeIdleState());
+
+  final QRCodeGetSeedUseCase useCase;
+
+  void getSeed() async {
```

Comment:
**suggestion**: make this as `Future<void>`

Reply:
I don't think we need this since I don't will wait for the cubit method, I'll wait for the state when the cubit emits one.